v1models/transform.go

```diff
@@ -272,7 +272,11 @@ func TransformProfileFromV1ToV2(profile DeviceProfile) (models.DeviceProfile, er
 	for i, cmd := range v2dp.DeviceCommands {
 		v2dp.DeviceCommands[i].IsHidden = isV2DeviceCommandHidden(cmd.Name, profile.CoreCommands)
 	}
-
+	// Convert StartingAddress for Modbus protocol
+	err = ConvertStartingAddressToZeroBased(&v2dp)
+	if err != nil {
+		return v2dp, errors.NewCommonEdgeXWrapper(err)
```

Comment:
Since we need the error message from backend on GUI to let user know that where is the error from the imported profile on DCT, can we modify the return error to
```
errors.NewCommonEdgeX(errors.KindContractInvalid, "convert startingAddress from string to int for v2 failed", err)
```

Reply:
Modified.